src/styles/index.scss

```diff
@@ -0,0 +1,5 @@
+@import './_default';
+@import './book';
```

Comment:
- [optional] This imported partial is causing a compile error since it's not yet a file.

Reply:
Thank you @FrederickMih! I will consider that